raiden/log_config.py

```diff
@@ -130,6 +130,27 @@ def processor_wrapper(msg: str) -> str:
     return processor_wrapper
 
 
+def configure_debug_logfile_path(debug_log_file_name: Optional[str]) -> str:
+    """Figure out the full pathname for the debug logfile depending on the user's OS"""
+    if debug_log_file_name is None:
```

Comment:
This doesn't seem right. 
Previously if a log file name is given it was used as is (it was assumed to be a fully qualified path).

This is also expected to work in at least the scenario player.

Reply:
Then the argument name and the argument description is wrong.

https://github.com/raiden-network/raiden/blob/acc44152354c65b0db0784efefad786f92fc7d26/raiden/ui/cli.py#L366-L370

 `Filename != Filepath` and from the code I expect it to be just a name. Since we write it in the current directory.

I am fine with whatever as long as the end result makes sense.

Reply:
So @ulope what do you think is the proper way to go here?

- rename the argument to `--debug-logfile-path`
- add logic to determine if it's a relative or an absolute path and treat it differently depending on what it is?

More?

Reply:
i guess changing the name of the option would be more explicit. If someone then passes a relative path and that blows up it's their own fault.

Reply:
Will do that and ping you when it's done then!

Reply:
Done @ulope. Check this again when you can.